Apps/GatewayApi/test/unit/Services.Test/UserProfileServiceTests.cs

```diff
@@ -885,72 +1089,22 @@ private static UpdateAcceptedTermsMock SetupUpdateAcceptedTermsMock(DbStatusCode
                     s => s.GetDataSourcesAsync(
                         It.Is<string>(x => x == Hdid),
                         It.IsAny<CancellationToken>()))
-                .ReturnsAsync(Enumerable.Empty<DataSource>());
+                .ReturnsAsync([]);
 
-            IUserProfileService service = GetUserProfileService(
+            return GetUserProfileService(
                 userProfileDelegateMock: userProfileDelegateMock,
                 legalAgreementServiceMock: legalAgreementServiceMock,
                 userPreferenceServiceMock: userPreferenceServiceMock,
                 applicationSettingsServiceMock: applicationSettingsServiceMock,
                 patientRepositoryMock: patientRepositoryMock);
-
-            RequestResult<UserProfileModel> expected = new()
-            {
-                ResourcePayload = updateResult.Status == DbStatusCode.Updated && userProfileExists
-                    ? new UserProfileModel
-                    {
-                        HdId = userProfile.HdId,
-                        TermsOfServiceId = TermsOfServiceGuid,
-                        AcceptedTermsOfService = true,
-                        HasTermsOfServiceUpdated = true,
-                        LastLoginDateTime = lastLoginDateTime,
-                        HasTourUpdated = false,
-                        BetaFeatures = [GatewayApi.Constants.BetaFeature.Salesforce],
-                    }
-                    : null,
-                ResultStatus = updateResult.Status != DbStatusCode.Updated || !userProfileExists
-                    ? ResultType.Error
-                    : ResultType.Success,
-                ResultError = updateResult.Status != DbStatusCode.Updated || !userProfileExists
-                    ? new()
-                    {
-                        ErrorCode = ErrorTranslator.ServiceError(ErrorType.CommunicationInternal, ServiceType.Database),
-                        ResultMessage = !userProfileExists ? "Unable to retrieve user profile" : "Unable to update the terms of service: DB Error",
-                    }
-                    : null,
-            };
-
-            return new(service, expected, Hdid, TermsOfServiceGuid);
         }
 
-        private static CloseUserProfileMock SetupCloseUerProfileMock(DbStatusCode updatedStatus, bool userProfileExists, bool profileClosed)
+        private static CloseUserProfileMock SetupCloseUerProfileMock(
```

Comment:
Existing typo: `SetupCloseUerProfileMock` should be `SetupCloseUserProfileMock`.